Prace_domowe/Praca_domowa1/Grupa1/Sebastian_Deregowski/PD1_Deregowski_Sebastian.ipynb

```diff
@@ -0,0 +1,2791 @@
+{
```

Comment:
<p>Zamiast przycinania wartości area do jakiejś arbitralnej wartości lepiej zastosować logarytmowanie z użyciem funkcji numpy.log1p()</p><p>Należy pamiętać o celu modelowania zmiennej <em>area</em> - to właśnie ekstremalnie duże pożary są najgroźniejsze i to właśnie je chcemy przewidywać. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>